.travis.yml

```diff
@@ -2,7 +2,7 @@
 # vi: ts=2 sw=2 et:
 
 language: bash
-dist: bionic
+dist: focal
 services:
   - docker
```

Comment:
this line should probably go as well

Reply:
I didn't want to mess with the docker involvement in the "regular" jobs since I wasn't sure if you've already started working on it. If not, I'll happily drop the docker support *everywhere* (in the Travis config).

Reply:
started looking but saw you were working on this, so was planning to wait, no point in working on the same infra at the same time :-)

is there any way I can help?

Reply:
> is there any way I can help?

So far there's no need, thanks :-). Right now everything looks like a huge mess because of how the encrypted tokens in Travis work, so I have to verify the Coverity functionality in [my fork](https://github.com/mrc0mmand/systemd/pull/7).